features/guidance/anticipate-lanes.feature

```diff
@@ -151,9 +151,9 @@ Feature: Turn Lane Guidance
             | cj    |                    | 1     | motorway_link | yes    | xbcj |
 
        When I route I should get
-            | waypoints | route        | turns                           | lanes                          |
-            | a,i       | ab,ci,ci     | depart,turn slight right,arrive | ,none:false slight right:true, |
-            | a,j       | ab,xbcj      | depart,arrive                   | ,                              |
+            | waypoints | route        | turns                           | lanes                           |
+            | a,i       | ab,ci,ci     | depart,turn slight right,arrive | ;,none:false slight right:true, |
+            | a,j       | ab,xbcj      | depart,arrive                   | ;;none:true slight right:false, |
```

Comment:
lanes do not change. We only show all lanes now. Before we were only showing lanes directly at the turns and suppressed use-lane as long as all lanes were allowed (as is the case here).